pkg/build/build.go

```diff
@@ -55,11 +55,27 @@ import (
 	"chainguard.dev/melange/pkg/sbom"
 )
 
+const melangeOutputDirName = "melange-out"
```

Comment:
is this rebased? as i think there is a new variable for this somewhere now. but maybe just in the substitution map.

Reply:
Yeah just rebased on the latest. I don't see this existing anywhere else, but if you spot it, let me know and I'll use that.